packages/web-worker/src/bugsnag.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-env worker, serviceworker */
+
+import delivery from '@bugsnag/delivery-fetch'
+import pluginClientIp from '@bugsnag/plugin-client-ip'
+import pluginWindowOnError from '@bugsnag/plugin-window-onerror'
+import pluginWindowUnhandledRejection from '@bugsnag/plugin-window-unhandled-rejection'
+
+// extend the base config schema with some browser-specific options
+import { schema as baseConfig } from '@bugsnag/core/config'
+import workerConfig from './config'
+import pluginBrowserDevice from '@bugsnag/plugin-browser-device'
+import pluginBrowserSession from '@bugsnag/plugin-browser-session'
+import pluginPreventDiscard from './prevent-discard'
+import ClientWithInternals from '@bugsnag/core/client'
+import type { Client, Config, BugsnagStatic } from '@bugsnag/core'
+import assign from '@bugsnag/core/lib/es-utils/assign'
+
+export interface WorkerConfig extends Config {
+  collectUserIp?: boolean
+  generateAnonymousId?: boolean
+}
+
+export interface WorkerBugsnagStatic extends BugsnagStatic {
+  start(apiKeyOrOpts: string | WorkerConfig): Client
+  createClient(apiKeyOrOpts: string | WorkerConfig): Client
+}
+
+const name = 'Bugsnag Web Worker'
+const url = 'https://github.com/bugsnag/bugsnag-js'
+// @ts-ignore
+const version = __BUGSNAG_NOTIFIER_VERSION__
```

Comment:
Not sure why but there seems to be a problem with this being `__VERSION__` specifically. If I try to use that rollup blows up with:

```
[!] RollupError: src/bugsnag.ts (14:25): Expected a semicolon (Note that you need plugins to import files that are not JavaScript)
src/bugsnag.ts (14:25)
12: var name = 'Bugsnag Web Worker';
13: var url = 'https://github.com/bugsnag/bugsnag-js';
14: var version = __VERSION__;
                             ^
15: var schema = assign({}, baseConfig, workerConfig);
```